modules/compute/gke-node-pool/variables.tf

```diff
@@ -171,10 +171,36 @@ variable "spot" {
   default     = false
 }
 
+# tflint-ignore: terraform_unused_declarations
 variable "compact_placement" {
-  description = "Places node pool's nodes in a closer physical proximity in order to reduce network latency between nodes."
+  description = "DEPRECATED: Use `placement_policy`"
```

Comment:
Is there a reason why we don't want to support backward compatibility for now? Also, do we know how frequently our users use this option? If it's fairly low, we should be good

Reply:
Yes, the usage is low. It was discussed offline.